.env.development

```diff
@@ -32,18 +32,8 @@ GOVUK_NOTIFY_API_KEY=not-a-real-key
 RUN_SIDEKIQ_IN_TEST_MODE=true
 ENABLE_SYNC_TRIGGER_ENDPOINT=true
 
-# App store credentials for testing against local app store
-#
-# - commenting out APP_STORE_TENANT_ID will prevent attempts to authenticate to configured app store.
-#   Note that the configured app store will require AUTHAUTHENTICATION_REQUIRED=false too in this case.
-#
-# - uncomment all and add secret values from k8s to autheticate
-#   Note that the configured app store must NOT have AUTHAUTHENTICATION_REQUIRED=false in this case.
-#
-APP_STORE_CLIENT_ID=not-a-real-key
-APP_STORE_TENANT_ID=not-a-real-key
-CASEWORKER_CLIENT_ID=not-a-real-key
-CASEWORKER_CLIENT_SECRET=not-a-real-key
+# App store authentication should not be required by default for local development
+AUTHENTICATION_REQUIRED=false
```

Comment:
As above, this isn't used anymore so can be removed

Reply:
done this also